web/apps/photos/src/services/machineLearning/mlWorkManager.ts

```diff
@@ -30,7 +30,7 @@ class MLWorkManager {
 
     constructor() {
         this.liveSyncQueue = new PQueue({
-            concurrency: 1,
+            concurrency: 4,
```

Comment:
This change doesn't come from me. I remember in the past there were some issues with concurrency, so we should be careful with this.